apps/graphql/package.json

```diff
@@ -38,7 +38,7 @@
     "xml2js": "^0.4.23"
   },
   "devDependencies": {
-    "eslint": "^7.1.0",
+    "eslint": "^6.8.0",
```

Comment:
Downgrading eslint?

Reply:
Ja, er is een conflict met andere dependencies die nog niet klaar zijn voor 7.x 😞 
Staat hier als reminder voor mezelf. https://github.com/Amsterdam/vergunningcheck/issues/81